package.json

```diff
@@ -23,9 +23,9 @@
     "lock": "npm run lint:versions -s && npm shrinkwrap --dev && sed -i '' 's/https:/http:/g' npm-shrinkwrap.json"
   },
   "dependencies": {
-    "jquery": "^2.2.4",
-    "uikit": "3.0.0-beta.23",
-    "throttle-debounce": "^1.0.1"
+    "jquery": "2.2.4",
+    "uikit": "3.0.0-beta.27",
+    "throttle-debounce": "1.0.1"
```

Comment:
I think using the `package-lock.json` file would be nice. I don't think we should jump on `npm@5` yet since the Node LTS is still at `npm@3`. If it is reasonable to jump to `npm@5` we'll just need to make sure the Dockerfiles in `cookie-monster` are updated and that the frontend team has it available as well.

In the interim, updating surface as uikit pushes releases isn't bad, we may even be able to set up one of those auto-pull-request bots to automate this.